src/main/java/teammates/storage/entity/FeedbackQuestion.java

```diff
@@ -51,6 +51,9 @@
     private int questionNumber;
     
     @Persistent
+    private Boolean isQuestionCompulsory;
```

Comment:
Actually, can you use a primitive `boolean` for this? It gets a default false value, so it's convenient for the existing legacy data too.